BLE_GattServer_ExperimentalServices/README.md

```diff
@@ -0,0 +1,56 @@
+# Link Loss Service
+
+## Introduction
+This application sends an alert message over a serial terminal once the connection to the client is lost without prior warning.
+The client sets the alert message string by writing a valid byte to a R/W GATT Characteristic inside the Link Loss Service (LLS):
+
+* 0x00 ("No Alert")
+* 0x01 ("Mild Alert") 
+* 0x02 ("High Alert")
+
+The alert ends after a user-defined timeout (= 1 minute).
+
+## Usage
+
+### Hardware Requirements
+Please refer to the main [README](https://github.com/ARMmbed/mbed-os-example-ble/blob/master/README.md).
+
+### Build Instructions
+Please refer to the main [README](https://github.com/ARMmbed/mbed-os-example-ble/blob/master/README.md).
+
+### Demostration
+**Note:** The steps given below demostrate the application using an Android smartphone running nRF Connect 4.24.3. 
+The illustations will be different for other clients.         
+1. Build the application and flash the board
+2. Open a serial terminal on your host to receive serial prints from the application:
+  `mbed term -b 115200`
+3. Open nRF Connect, locate your device and press connect (Fig. 1)
+
+  <p align="center">
+    <img height="280" src="https://github.com/noonfom/mbed-os-example-ble/blob/experimental-services/BLE_GattServer_ExperimentalServices/img/scan.png">
```

Comment:
cannot link to your repo, please use relative path

same for similar occurrences below